build.gradle

```diff
@@ -48,6 +50,14 @@ subprojects {
 
     compileJava.options.compilerArgs << '-Xlint:-options' // remove warning about bootstrap class path
 
+
+    audit {
+         failOnError = false
+         rateLimitAsError = false
+         cache = "/tmp/ossindex.cache"
```

Comment:
careful this is not cross-platform.
You maybe can get the tmp directory from gradle or using a java property

Reply:
Comment addressed